src/structurizr/model/container.py

```diff
@@ -15,6 +15,7 @@
 
 """Provide a container model."""
 
+from __future__ import annotations
```

Comment:
This doesn't work in Python 3.6.  Before I work around, are we still planning to support 3.6?

Reply:
Actually, I've taken the incompatible stuff out for the moment so I don't clutter the PRs.